ietf/ipr/tests.py

```diff
@@ -192,6 +193,38 @@ def test_search(self):
         r = self.client.get(url + "?submit=rfc&rfc=321")
         self.assertContains(r, ipr.title)
 
+        rfc_new = RfcFactory(rfc_number=322)
+        rfc_new.relateddocument_set.create(relationship_id="obs", target=rfc)
+
+        # find RFC 322 which obsoletes RFC 321 whose draft has IPR
+        r = self.client.get(url + "?submit=rfc&rfc=322")
+        self.assertContains(r, ipr.title)
+        self.assertContains(r, "Total number of IPR disclosures found: <b>1</b>")
+        self.assertContains(r, "Total number of documents searched: <b>3</b>.")
+        self.assertContains(
+            r,
+            f"""Results for <a href="/doc/{rfc_new.name}/">{prettify_std_name(rfc_new.name)}</a>
+                        ("{rfc_new.title}")""",
+        )
+        self.assertContains(
+            r,
+            f"""Results for <a href="/doc/{rfc.name}/">{prettify_std_name(rfc.name)}</a>
+                        ("{rfc.title}"), which
+                            
+                                was obsoleted by
+                                <a href="/doc/{rfc_new.name}/">{prettify_std_name(rfc_new.name)}</a>
+                                ("{rfc_new.title}")""",
+        )
+        self.assertContains(
+            r,
+            f"""Results for <a href="/doc/{draft.name}/">{prettify_std_name(draft.name)}</a>
+                        ("{draft.title}"), which
+                            
+                                became rfc
+                                <a href="/doc/{rfc.name}/">{prettify_std_name(rfc.name)}</a>
+                                ("{rfc.title}")""",
+        )
+
```

Comment:
Having a test live at risk of failing because of whitespace change in the html isn't good.
I'll let this in, but we should change this soon to something that is testing for the important content and not the layout details. Looking just to see that the right drafts and RFCs are mentioned should be enough.